lxc/exec.go

```diff
@@ -202,10 +202,14 @@ func (c *cmdExec) Run(cmd *cobra.Command, args []string) error {
 	}
 
 	// Wait for the operation to complete
-	err = op.Wait()
```

Comment:
@markylaing so is this the call that fails/blocks forever (which is it btw)?

Reply:
Yes it fails here 
https://github.com/canonical/lxd/blob/b9ec9c1e3b8bb87cb4d65f0087a80bfed410c3be/client/operations.go#L128-L131
with a 403